app/assets/stylesheets/custom.scss

```diff
@@ -25,7 +25,7 @@ h6 {
 }
 
 table a, a:hover{
-  color: #A5C6DE;
+  color: #1F75FE;
```

Comment:
Color literals like `#1F75FE` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#1F75FE` should be written as `#1f75fe`